src/Swagger/ApiDocumentationBuilder.php

```diff
@@ -46,11 +46,10 @@
     private $description;
     private $iriConverter;
     private $version;
-    private $host;
-    private $schema;
+    private $formats;
     const SWAGGER_VERSION = '2.0';
```

Comment:
According to our CS constants must be before properties (and add a blank line between constants an property).